content/enterprise/admin/infrastructure/demo-to-disk-migration.mdx

```diff
@@ -0,0 +1,147 @@
+
+```
+Unmet start requirement: Required setting "Mounted Disk Path" is not set.
+```
+
+If you run `replicatedctl app status`, you will see output similar to the
+following.
+
+```json
+[
+    {
+        "AppID": "f9486004126d40904381e029d76353d0",
+        "Sequence": 609,
+        "PatchSequence": 0,
+        "State": "stopped",
+        "DesiredState": "started",
+        "Error": "Unmet start requirement: Required setting \"Mounted Disk Path\" is not set.",
+        "IsCancellable": false,
+        "IsTransitioning": false,
+        "LastModifiedAt": "2022-04-11T21:02:18.613686589Z"
+    }
+]
+```
+
+The Replicated console shows the following page.
+
+![A screenshot showing the error message on the Replicated console.](/img/docs/demo-to-disk.png)
+
+## Migrating Application Data From Demo Mode to Mounted Disk Mode
+
+1. Connect to your Terraform Enterprise instance.
+
+1. Create the Mounted Disk directory. You can choose any directory, but Terraform Enterprise expects that the directory is backed by persistent storage that can be mounted to the instance.
+
+  ```bash
+  mkdir -p /opt/terraform-enterprise
+  ```
+
+1. Set the `disk_path` application setting to the mounted disk directory.
+
+  ```bash
+  replicatedctl app-config set disk_path --value '/opt/terraform-enterprise'
+  ```
+
+1. Migrate the PostgreSQL data to the mounted disk directory.
+
+    1. Create the `postgres` directory under the mounted disk directory.
+
+      ```bash
+      mkdir -p /opt/terraform-enterprise/postgres
+      ```
+
+    1. Retrieve the PostgreSQL data path from the Docker volume.
+
+      ```bash
+      demo_database="$(docker volume inspect postgres --format '{{ .Mountpoint }}')"
+      ```
+
+    1. Migrate the PostgreSQL data to the mounted disk directory.
+
+      ```bash
+      cp -r "${demo_database}"/* /opt/terraform-enterprise/postgres
+      ```
+
+    1. Mark the PostgreSQL data as migrated.
+
+      ```bash
+      touch /opt/terraform-enterprise/postgres/.poc-to-disk-migrated
+      ```
+
+1. Migrate the object storage data to the mounted disk directory.
+
+    1. Create the `aux` directory under the mounted disk directory.
+
+      ```bash
+      mkdir -p /opt/terraform-enterprise/aux
+      ```
+
+    1. Retrieve the object storage data path from the Docker volume.
+
+      ```bash
+      demo_object_storage="$(docker volume inspect aux --format '{{ .Mountpoint }}')"
+      ```
+
+    1. Migrate the object storage data to the mounted disk directory.
+
+      ```bash
+      cp -r "${demo_object_storage}"/* /opt/terraform-enterprise/aux
+      ```
+
+    1. Mark the object storage data as migrated.
+
+      ```bash
+      touch /opt/terraform-enterprise/aux/.poc-to-disk-migrated
+      ```
+
+1. Restart Terraform Enterprise to apply the changes and start the application in _Mounted Disk_ mode.
+
+    1. Stop Terraform Enterprise.
+
+      ```bash
+      replicatedctl app stop
+      ```
+
+    1. Wait for Terraform Enterprise to stop.
```

Comment:
Should we tell them what the status will be or is it like super duper obvious? I noticed you asked them to run this status command so I was thinking maybe we needed them to wait for something specific --> I guess even just for it to say "Stopped" but idk should we say that or tell them it takes a few minutes?

Reply:
The output would look like this:

```
[
    {
        "AppID": "8e22d0e4b96b448a5625b5a232132782",
        "Sequence": 609,
        "PatchSequence": 0,
        "State": "stopped",
        "DesiredState": "stopped",
        "Error": "",
        "IsCancellable": false,
        "IsTransitioning": false,
        "LastModifiedAt": "2022-04-18T19:46:12.924441325Z"
    }
]
```

I updated the line to this:

```
1. Wait for Terraform Enterprise to stop. Terraform Enterprise is stopped once the `State` shows `stopped`.
```

You can never be too verbose.